frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -1,46 +1,34 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import {
-  AppBar as MuiAppBar,
-  Box,
-  Divider as MuiDivider,
-  IconButton,
-  Toolbar,
-  Typography,
-} from "@material-ui/core";
+import styled from "@emotion/styled";
+import { AppBar as MuiAppBar, Box, Grid, IconButton, Toolbar, Typography } from "@material-ui/core";
 import MenuIcon from "@material-ui/icons/Menu";
-import styled from "styled-components";
 
 import Drawer from "./drawer";
 import Logo from "./logo";
+import Notifications from "./notifications";
 import SearchField from "./search";
 import { UserInformation } from "./user";
 
-const AppBar = styled(MuiAppBar)`
-  ${({ theme }) => `
-  min-width: fit-content;
-  background-color: ${theme.palette.secondary.main};
-  color: ${theme.palette.primary.main};
-  min-width: fit-content;
-  `}
-`;
+// TODO (sperry): make header responsive for small devices
+const AppBar = styled(MuiAppBar)({
+  minWidth: "fit-content",
+  background: "linear-gradient(90deg, #38106b 4.58%, #131c5f 89.31%)",
+});
 
-const MenuButton = styled(IconButton)`
-  padding: 12px;
-  margin-left: -12px;
-`;
+const MenuButton = styled(IconButton)({
+  padding: "12px",
+  marginLeft: "-12px",
+});
 
-const Title = styled(Typography)`
-  margin-right: 25px;
-  font-weight: bolder;
-`;
-
-const Divider = styled(MuiDivider)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.primary.main};
-  margin: 16px 8px;
-  `}
-`;
+// TODO (sperry): remove marginRight in new search bar design
+const Title = styled(Typography)({
+  margin: "12px 25px 12px 8px",
+  fontWeight: "bold",
+  fontSize: "20px",
+  color: "#ffffff",
+  opacity: "0.87",
```

Comment:
make opacity part of the font color

```suggestion
  color: "rgba(255, 255, 255, 0.87)",
```